.ci/jenkins/dsl/jobs.groovy

```diff
@@ -253,14 +254,15 @@ void setupPostReleaseJob() {
     }
 }
 
-void setupOptaPlannerTurtleTestsJob() {
-    def jobParams = KogitoJobUtils.getBasicJobParams(this, 'optaplanner-turtle-tests', Folder.OTHER, "${jenkins_path}/Jenkinsfile.turtle",
-            'Run OptaPlanner turtle tests on a weekly basis.')
+void setupOptaPlannerTurtleTestsJob(String csType) {
+    def jobParams = KogitoJobUtils.getBasicJobParams(this, "optaplanner-turtle-tests-${csType}", Folder.OTHER, "${jenkins_path}/Jenkinsfile.turtle",
+            "Run OptaPlanner turtle tests with CS-${csType} on a weekly basis.")
     jobParams.triggers = [ cron : 'H H * * 5' ] // Run every Friday.
     KogitoJobTemplate.createPipelineJob(this, jobParams)?.with {
         parameters {
             stringParam('BUILD_BRANCH_NAME', "${GIT_BRANCH}", 'Git branch to checkout')
             stringParam('GIT_AUTHOR', "${GIT_AUTHOR_NAME}", 'Git author or an organization.')
+            stringParam('CS_TYPE', "${csType}", 'CS implementation type.')
```

Comment:
I guess this should be an env instead of a param as you will not change it or ?

Reply:
Yes, that would be probably better; let me have a look.